controller/controller.go

```diff
@@ -36,10 +41,10 @@ const (
 )
 
 type Config struct {
-	CRD       *apiextensionsv1beta1.CustomResourceDefinition
-	CRDClient *k8scrdclient.CRDClient
-	Informer  informer.Interface
-	Logger    micrologger.Logger
+	CRD            *apiextensionsv1beta1.CustomResourceDefinition
+	BackOffFactory func() backoff.Interface
+	K8sClient      k8sclient.Interface
```

Comment:
I would add a godoc reminding about setting `AddToScheme ` for custom resources reconciliation.

Reply:
Sorted.